### PR TITLE
chore: Add code fences to samples

### PR DESCRIPTION
**Pull request by nschonni**
Because ✨

### CODE REVIEW

**Comment by PRMerger9** [comment]
@nschonni : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

**Comment by TerryGLee** [comment]
@nschonni thanks for taking the time to put together this PR.
@gewarren please take a look? many of these edits affect content in your purview. (The suggested edit to the VSTO topic is fine.)

**Comment by PRMerger7** [comment]
@nschonni : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@nschonni : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@nschonni : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

File: docs/modeling/how-to-extend-the-domain-specific-language-designer.md [hunk 1]
```diff
@@ -82,8 +82,6 @@ To uninstall the extension, follow these steps:
 
 To make a shortcut menu command appear on the DSL Designer surface or in the DSL Explorer window, write a class resembling the following.
 
-The class must implement `ICommandExtension` and must have the attribute `DslDefinitionModelCommandExtension`.
```
Comment (gewarren) [inline docs/modeling/how-to-extend-the-domain-specific-language-designer.md]: Why were these three lines deleted in this file?

Comment (nschonni) [reply docs/modeling/how-to-extend-the-domain-specific-language-designer.md]: My guess would be from a rebase rather than intentional

Comment (nschonni) [reply docs/modeling/how-to-extend-the-domain-specific-language-designer.md]: Looks like the rebase got confused with https://github.com/MicrosoftDocs/visualstudio-docs/commit/53fa78032c0272147929db3d0d10cb11a80543a8
I did this a few weeks ago and forgot, then rebased yesterday

**Comment by PRMerger6** [comment]
@nschonni : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

**Comment by nschonni** [comment]
@gewarren I've rebased out the rebase problems 😝

**Pull request merged**